src/content/docs/style-guide/structure/callouts.mdx

```diff
@@ -85,9 +85,10 @@ Here are examples of our callout variants:
 
       For preview docs, add a custom callout. Also, we recommend that you don't include a variant so you display the default color:
 
-      <Callout title="PREVIEW FEATURE">
-        This feature is still in development, but we encourage you to try it out!
-      </Callout>
+        <Callout title="preview">
```

Comment:
```suggestion
        <Callout title="Preview">
```

Reply:
Make sure to change all the other references in this PR

Reply:
Thanks @ally-sassman. But then should I change all the callout titles throughout the docs site? The other callout types are these:
```
<Callout variant="tip">
<Callout variant="important">
<Callout variant="caution">
```

Reply:
Ah, disregard this! I thought it wouldn't automatically uppercase it since it's a `title=` and not `variant=` (if you create a custom title, you have to specify if you want it uppercased or not). But looking at the preview link, it looks like it renders it in all caps (`PREVIEW`).

Reply:
Thank you, @ally-sassman!